spec.bs

```diff
@@ -92,12 +92,54 @@ table {
   top: -0.8em;
   left: -0.8em;
 }
+
+/* props from https://resources.whatwg.org/standard.css */
+dl.props { display: grid; grid-template-columns: max-content auto; row-gap: 0.25em; column-gap: 1em; }
+dl.props > dt { grid-column-start: 1; margin: 0; }
+dl.props > dd { grid-column-start: 2; margin: 0; }
+p + dl.props { margin-top: -0.5em; }
 </style>
 
 <script src="https://resources.whatwg.org/file-issue.js" async></script>
 
 <h2 id=urlpattern-class>The {{URLPattern}} class </h2>
 
+A {{URLPattern}} consists of several [=components=], each of which represents a [=/pattern string|pattern=] which could be matched against the corresponding component of a [=/URL=].
+
+It can be constructed using a string for each component, or from a shorthand string. It can optionally be resolved relative to a base URL.
+
+<div class="example">
+  The shorthand "`https://example.com/:category/*`" corresponds to the following components:
+
+  <dl class="props">
+    :  [=protocol component|protocol=]
```

Comment:
QQ: I'm still not familiar with the spec language but we don't have to write `=URLPattern/protocol component` here? Same questions to other components.

Reply:
It's not required (because it's unambiguous) but I can certainly do that.

Reply:
Got it, thanks you!